src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -231,7 +231,9 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, url }) => {
 	);
 
 	// spaces
-	const { data: accessibleVisitorSpaces } = useGetAccessibleVisitorSpaces();
+	const { data: accessibleVisitorSpaces } = useGetAccessibleVisitorSpaces({
+		canViewAllSpaces: user?.permissions.includes(Permission.READ_ALL_SPACES) ?? false,
+	});
```

Comment:
You can use the existing `useHasAllPermission` hook ;) 

```suggestion
        const canViewAllSpaces = useHasAllPermission(Permission.READ_ALL_SPACES);
	const { data: accessibleVisitorSpaces } = useGetAccessibleVisitorSpaces({
		canViewAllSpaces
	});
```